src/components/sidebar/SafeListItem/styles.module.css

```diff
@@ -2,19 +2,61 @@
   margin-top: var(--space-1);
   margin-bottom: var(--space-1);
   padding-left: var(--space-2);
-  padding-right: var(--space-2);
 }
 
 .safe {
   border-radius: 8px;
+  border: 1px solid var(--color-border-light);
 }
 
 .open {
   padding-left: 16px;
+  border: 1px solid var(--color-secondary-light);
   border-left: 6px solid var(--color-secondary-light);
   background-color: var(--color-secondary-background) !important;
 }
 
 .safe:hover {
   background-color: var(--color-secondary-background);
 }
+
+.pendingButtons {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 4px;
+  padding-left: 0px;
+  margin-right:  var(--space-2);
+}
+
+.pendingButton {
+  width: 52px;
+  height: 22px;
+  border-top-right-radius: 6px;
+  border-bottom-right-radius: 6px;
```

Comment:
We should use the theme's `shape.borderRadius` here instead. You will have to apply it inside of an `sx` prop.